examples/gocean/eg5/readonly/Makefile

```diff
@@ -42,17 +42,18 @@
 # The read-only wrapper library:
 #     export READ_ONLY_DIR = ../../../lib/read_only/dl_esm_inf
 
-include ../../common.mk
+PSYROOT=../../../..
```

Comment:
You use `$(realpath ...)` in other places. Should that be here too?

Reply:
No, while it can't hurt, it is not required. The problem only occurs when you provide a variable relative to directory ``x``, when calling a makefile in directory ``y``. E.g.: drhook and dl_timer, which (by default) were set to be relative to the current directory. This worked in the past by cheer coincidence: it was the same "number of directories": ``.../psyclone/examples/gocean/eg5`` and ``.../psyclone/lib/profile/drhook`` - any reference starting with ``../../..`` (which brings us back to the psyclone root directory) works in both directories. But after I moved ``.../gocean/eg5`` to ``.../gocean/eg5/profile``, the levels were different, and the reference to drhook and dl_timer which were relative to ``.../gocean/eg5/profile`` did not work in ``.../psyclone/lib/profile/drhook`` (etc) anymore. That's why I had to supply an absolute path (and also notice, if the user overwrites the default location, the value should be an absolute path, but that's not really specific to this PR).

Reply:
Aha. I live and learn :-)